migration/src/main/scala/db/migration/V003__UpdateUsers.scala

```diff
@@ -0,0 +1,20 @@
+package db.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import java.sql.PreparedStatement
+
+
+class V003__UpdateUsers extends BaseJavaMigration{
+
+
+  @throws[Exception]
+  override def migrate(context: Context): Unit = {
+    try {
+      val statement = context.getConnection.prepareStatement("INSERT INTO test_user (name) VALUES ('Obelix')")
+      try statement.execute
+      finally if (statement != null) statement.close()
+    }
+  }
```

Comment:
That's great that we can have a Java/Scala file to do the migration. This will be useful for the unwrapping of the payload and for re-writing the tags. 

Also, when a user wants to re-shard the tags, they can add a class that redefines the sharding number.

Reply:
Code-based migrations in Flyway are my all-time favorite feature.